### PR TITLE
list environment groups for a given model

### PR DESCRIPTION
**Pull request by eecaro**

### CODE REVIEW

File: tests/test_integration.py [hunk 1]
```diff
@@ -47,6 +47,15 @@ def test_list_versions_empty(self):
         assert isinstance(res, list)
         assert len(res) == 0
 
+    def test_list_environment_groups(self):
+        res = models.list_environment_groups(
+            model_name=TESTING_MODEL,
+            **PARAMS
+        )
+        # Check response type and verfiy that at least dev and prod environment groups come back from testing app/model
```
Comment (tylerhutcherson) [inline tests/test_integration.py]: maybe explicitly check that the values in the list contain the names dev and prod too

File: skafos/models.py [hunk 2]
```diff
@@ -358,3 +358,51 @@ def list_versions(**kwargs) -> list:
 
     versions = _clean_up_version_list(res)
     return versions
+
+def list_environment_groups(**kwargs) -> list:
```
Comment (tylerhutcherson) [inline skafos/models.py]: would we prefer just `list_environments`? or something similar? We don't refer to them as environment groups anywhere in the docs, but that could change too

File: tests/test_integration.py (hunk 1 above)
Comment (eecaro) [reply tests/test_integration.py]: Sounds good, I'll make that change.

File: skafos/models.py (hunk 2 above)
Comment (eecaro) [reply skafos/models.py]: @tylerhutcherson That's a great call out. I'm conflicted about this one. `list_environments` is more clear to our users at this moment, but `list_environment_groups` encapsulates the true functionality and prepares them for the terminology. There are also backwards compatibility and breaking change considerations. @kmussel what's your take on `list_environments` vs. `list_environment_groups`?

**Pull request merged**